pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -187,7 +188,8 @@ func (m *managerImpl) Start(diskInfoProvider DiskInfoProvider, podFunc ActivePod
 		klog.InfoS(message)
 		m.synchronize(diskInfoProvider, podFunc)
 	}
-	if m.config.KernelMemcgNotification {
+	klog.InfoS("Eviction manager: starting control loop")
+	if m.config.KernelMemcgNotification || runtime.GOOS == "windows" {
```

Comment:
what was the behavior before when people tried to set `KernelMemcgNotification` to true? I would think it will fail kubelet?